### ISSUE TITLE
Duplicate API request?

### ISSUE THREAD

**Issue by zvrh** [bitwarden/web#1324]
### Steps To Reproduce

1. Open browser developer tools network view
2. Go to '<https://vault.bitwarden.com/#/vault>'
3. Click on 'Log in'

![20211207163323](https://user-images.githubusercontent.com/16538800/144994525-1e86917b-417e-4056-8110-c8c03835867b.png)

### Expected Result

<https://vault.bitwarden.com/identity/connect/token> and <https://vault.bitwarden.com/api/sync?excludeDomains=true> request only once

### Actual Result

The same API request was sent twice

### Screenshots or Videos

![20211207163323](https://user-images.githubusercontent.com/16538800/144994525-1e86917b-417e-4056-8110-c8c03835867b.png)

### Operating System

Windows

### Web Browser

Chrome

### Browser Version

96.0.4664.45 (64bit)

### Build Version

2.24.4

**Comment by BlackDex** [comment]
I see the same issue, would be nice if this could be solved. Especially for large vaults this could have a big impact.
I did some investigation on this, and it looks like this sneaked in probably somewhere here: https://github.com/bitwarden/jslib/compare/18bf616e2e0821a46b1cb71737d80b099a64b237...0a2ff12bedd55f5982fb8fe9102c4f20522f72e1#diff-0c52bd15045fcce50e05b948e322514b5bda1bcaed955d1524df77d1e12c4ad4

Since this bug isn't present in v2.21.1 but is in v2.22.0: https://github.com/bitwarden/web/compare/ae091bd738f334e3bc3296091247992330be3e92...v2.22.0

I tried to look for this my self, but i do not have enough knowledge about TypeScript to find this easily.
I did found the double token request tho i think, it is regarding this change in `common/src/services/sync.service.ts` on line 97.
If i comment out that specific line in the current v2.26.1 version that double token request is gone.

Again, I do not have any good TypeScript knowledge. I just hope this will point the devs into a direction on how to solve this.
